sdks/python/tests/library_integration/openai/test_evaluation_metrics.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+
+from opik.evaluation import metrics
+
+
+def assert_score_result(result):
+    assert isinstance(result.value, float)
+    assert 0.0 <= result.value <= 1.0
+    assert isinstance(result.reason, str)
+    assert len(result.reason) > 0
+
+
+@pytest.mark.parametrize(
+    argnames="context",
+    argvalues=[
+        None,
+        ["France is a country in Europe."],
+    ],
+)
+def test__answer_relevance(context):
```

Comment:
each of these tests should have `ensure_openai_configured` fixture used (FYI in my current PR I moved it to library_integration.conftest)